test/test_helper.rb

```diff
@@ -8,6 +8,7 @@
 
 require 'action_dispatch'
 
+require 'responders'
```

Comment:
Not sure why this is necessary (there's a `Bundler.require` call a few lines above), but I needed this to make `bundle exec rake test` work locally.

Reply:
But the Gemfile includes the gemspec?

Reply:
But only like `gem 'decorate-responder', path: '.'`. It still will *not* load gems runtime dependencies.